ios/share/Info.plist

```diff
@@ -18,7 +18,7 @@
 	<key>HostAppBundleIdentifier</key>
 	<string>com.chat.expensify.chat</string>
 	<key>HostAppURLScheme</key>
-	<string>new-expensify</string>
+	<string>new-expensify://</string>
```

Comment:
Was this change necessary? I thought I remembered it working without that

Reply:
Yeah this is super confusing. This one needs the `://`. In the [README](https://github.com/Expensify/react-native-share-menu/blob/master/IOS_INSTRUCTIONS.md#create-app-group-and-url-scheme) they explicitly mention not including it for `CFBundleURLSchemes` but including it for `HostAppURLScheme`. Seems like it wouldn't be too hard to update the code to accept both. I'm also curious if there's a way to get the host app url scheme from native avoiding this altogether.